packages/grid/x-data-grid-pro/src/tests/events.DataGridPro.test.tsx

```diff
@@ -330,6 +331,148 @@ describe('<DataGridPro /> - Events Params', () => {
     });
   });
 
+  describe('gridReady', () => {
```

Comment:
@yared can you move the tests inside the `x-data-grid` folder? This will simplify the process going forward and no CLA will be needed. Also, it makes more sense since the `gridReady` event is not a pro or premium feature.
You can create a new file here `./packages/grid/x-data-grid/src/tests/events.DataGrid.test.tsx`